charts/vh-bookings-api/values.yaml

```diff
@@ -16,26 +16,28 @@ java:
       resourceGroup: vh-infra-core-{{ .Values.global.environment }}
       usePodIdentity: true
       secrets:
-        - servicebusqueue--connectionstring
-        - connectionstrings--vhbookings
         - applicationinsights--instrumentationkey
+        - azuread--tenantid
+        - connectionstrings--vhbookings
+        - servicebusqueue--connectionstring
     vh-bookings-api:
       excludeEnvironmentSuffix: false
       resourceGroup: vh-infra-core-{{ .Values.global.environment }}
       usePodIdentity: true
       secrets:
-        - azuread--clientsecret
         - azuread--clientid
-        - KinlyConfiguration--SipAddressStem
-        - FeatureFlags--StaffMemberFeature
-        - FeatureFlags--EJudFeature
+        - azuread--clientsecret
+        - kinlyconfiguration--sipaddressstem
+        - launchdarkly--sdkkey
   environment:
     ASPNETCORE_URLS: http://+:8080
-    AZUREAD__TENANTID: "fb6e0e22-0da3-4c35-972a-9d61eb256508"
+    AZUREAD__BOOKINGSAPIRESOURCEID: http://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net
```

Comment:
I would get this from the key vault as it is set in Terraform.
e.g. https://github.com/hmcts/vh-scheduler-jobs/blob/master/charts/vh-scheduler-jobs/values.yaml
https://github.com/hmcts/vh-scheduler-jobs/blob/a742904962c03d0b7428fd25cf1459db1b9c85a9/SchedulerJobs/SchedulerJobs/Startup.cs#L59

Reply:
resolved.